Modules/Registration/Common/test/itkMultiResolutionImageRegistrationMethodTest.cxx

```diff
@@ -135,7 +135,8 @@ itkMultiResolutionImageRegistrationMethodTest(int, char *[])
   registration->SetInitialTransformParameters(initialParameters);
   ITK_TEST_SET_GET_VALUE(initialParameters, registration->GetInitialTransformParameters());
 
-  ParametersType initialTransformParametersOfNextLevel(1);
+  typename ParametersType::ValueType initialTransformParametersOfNextLevelVal(0.0);
```

Comment:
`typename ParametersType::ValueType`...? Interesting! Before C++11, such a use of `typename` was disallowed, because it is outside of a template. With C++11 and C++14, it should just compile fine  😃 

Only a very _very_ old compiler may complain, like GCC 4.4 at https://godbolt.org/z/xsP94bndW saying _error: using 'typename' outside of template_ Fortunately we do not need to support GCC 4.4 anymore  😄  So no problem!